### PR TITLE
Add hawkular admin cluster role to management admin

### PR DESCRIPTION
**Pull request by simon3z**
cc @mwringe @detiber

### CODE REVIEW

File: roles/openshift_manageiq/vars/main.yml [hunk 1]
```diff
@@ -9,6 +9,20 @@ manageiq_cluster_role:
       verbs:
       - '*'
 
+manageiq_metrics_admin_clusterrole:
```
Comment (simon3z) [inline roles/openshift_manageiq/vars/main.yml]: @mwringe @detiber not sure if this belongs here... also how do you remember to keep this in sync these with `origin-metrics`?

**Comment by detiber** [comment]
@sdodson ptal

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
@simon3z which releases do you need this in? 3.4 and 3.3?

**Comment by simon3z** [comment]
> @simon3z which releases do you need this in? 3.4 and 3.3?

@sdodson only 3.4 thanks!

**Pull request merged**